### DISCUSSION TITLE
I can't carry my wallet

### DISCUSSION THREAD

**Discussion by ramonrogaciano** [Chia-Network/chia-blockchain#5915] [Support]
Yesterday I had a problem trying to create 2 batches, the disk got full and gave error. I put to delete the lot but as it was taking time, I closed the chia and restarted the pc. But after that it doesn't carry my wallet anymore, just keep on trying to connect. Ja desisntalei and reinstalled, already restored recovery point on pc, i have deleted the folders and reinstalled, i have already cleaned the temp, the %temp% and the prefetch and did not work and lastly now, I restored my pc to factory mode and even then not carries the wallet. thought it might be some blocking that the owner did on the modem, but I tested it on different network and it didn't work and tested it on my note and loaded it. but on the pc where I have my disks does not carry the wallet. already installed the version 15 also and did not work.

![erro chia](https://user-images.githubusercontent.com/84696953/119296486-021af680-bc27-11eb-866c-644b18b4ffce.jpg)

**Comment by MaximusChia** [comment]
I had similar problem, when my faulty RAM made me crash my system while plotting. I did a clean install of windows. If i conserve personal data Chia Client still stuck on connecting wallet. SO I did a clean install and wiping everything data. Then It got restored, but had to resync all the blockchain.

**Reply by ramonrogaciano** [reply]
I am reluctant not to have to do this, but I see that it may be the only solution. thanks for the comment and help

**Comment by BrandtH22** [comment]
@ramonrogaciano , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
